test/runtests.jl

```diff
@@ -91,6 +91,16 @@ prepend!(tests, linalg_tests)
 
 import LinearAlgebra
 cd(@__DIR__) do
+    # `net_on` implies that we have access to the loopback interface which is
+    # necessary for multithreading. There are some build environments that do
+    # not allow access to loopback, so we must disable multithreading when
```

Comment:
```suggestion
    # necessary for Distributed multi-processing. There are some test environments that do
    # not allow access to loopback, so we must disable addprocs when
```

Keno appears to be correct that we could also instead bind to a random file name (https://man7.org/linux/man-pages/man7/network_namespaces.7.html). It looks like most of the code already would accept a file path, which could be handled by checking `isabspath(bind_addr)`